src/hud_elements.cpp

```diff
@@ -1196,6 +1203,78 @@ void HudElements::device_battery()
 #endif
 }
 
+void HudElements::liquid_stats()
```

Comment:
Since liquid is only for unix and we only compile liquid.cpp on unix, the windows compiler does not have access to liquidStats.
set `#ifndef _WIN32` inside this function so it's not being compiled with mingw.
This should resolve the failing mingw test